defaults/main.yml

```diff
@@ -47,15 +47,26 @@ openvpn_cipher: AES-256-CBC
 openvpn_tun_mtu:
 openvpn_keepalive_ping: 5
 openvpn_keepalive_timeout: 30
-openvpn_service_user: nobody
-openvpn_service_group: nogroup
+openvpn_service_user: openvpn
+openvpn_service_group: openvpn
```

Comment:
I suppose the distributed rpm does say it should create the user & group: https://src.fedoraproject.org/rpms/openvpn/blob/rawhide/f/openvpn.spec

I do wonder if this will break existing installs though. Would it be possible to set this only if `openvpn_use_otp` is true?